community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -22,16 +22,24 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Create install directory
+    ansible.builtin.command:
+      mkdir -p "{{ install_dir | dirname }}"
+    changed_when: false
+
   - name: Acquire lock
-    ansible.builtin.shell:
-      mkdir -p "{{ install_dir | dirname }}/.ramble_lock"
+    ansible.builtin.command:
+      mkdir "{{ install_dir | dirname }}/.ramble_lock"
     register: lock_out
+    changed_when: lock_out.rc == 0
+    failed_when: false
 
   - name: Clones ramble into installation directory
     ansible.builtin.git:
       repo: "{{ ramble_url }}"
       dest: "{{ install_dir }}"
       version: "{{ ramble_ref }}"
+      force: true
```

Comment:
I had thought you said this was not necessary.

Reply:
Good catch.